### PR TITLE
Update example and document

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
## Why

Fixes: Document update and samples on #662 and #663.

## What

- Updated sample scripts
- Added metrics plugin extension to `Examples.Vendor.Distro`
- Small update to examples readme

## Checklist

- [ ] ~~`CHANGELOG.md` is updated.~~
- [X] Documentation is updated.
- [ ] ~~New features are covered by tests.~~

### CODE REVIEW

File: run-example.sh [hunk 1]
```diff
@@ -58,13 +59,13 @@ trap finish EXIT
...
-unset OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS
+export {OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS,OTEL_DOTNET_AUTO_METRICS_PLUGINS}="Examples.AspNetCoreMvc.OtelSdkPlugin, Examples.AspNetCoreMvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null:Examples.Vendor.Distro.Plugin, Examples.Vendor.Distro, Version=0.0.1.0, Culture=neutral, PublicKeyToken=null"
+ENABLE_PROFILING=${enableProfiling} OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS="AspNet,MongoDb,SqlClient" OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES="MyCompany.MyProduct.MyLibrary" OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS="NetRuntime" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${tracesexporter} OTEL_METRICS_EXPORTER=${metricsexporter} ./dev/instrument.sh ASPNETCORE_URLS="http://127.0.0.1:8080/" dotnet ./examples/AspNetCoreMvc/bin/${configuration}/${aspNetAppTargetFramework}/Examples.AspNetCoreMvc.dll &
+unset OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS OTEL_DOTNET_AUTO_METRICS_PLUGINS
```
Comment (pjanotti) [inline run-example.sh:64]: This makes me think that we should rename `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS` to `OTEL_DOTNET_AUTO_TRACES_PLUGINS`. However, that it is unrelated to this change.

**Review by pjanotti — COMMENTED** [review]
LGTM

**Review by pjanotti — APPROVED** [review]

File: run-example.sh (hunk 1 above)
Comment (rajkumar-rangaraj) [reply run-example.sh:64]: Created #835 to rename after merging this PR.

File: examples/README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ You can customize the script using the following environment variables:
 | **exampleApp**                  | Application selected for the `http-client` service                                  | `ConsoleApp`                                                                  |
 | **exampleAppInjectSDK**         | Whether automatic instrumentation injects the SDK on the `http-client` service      | `true`                                                                        |
 | **exampleAppTargetFramework**   | Target framework for the `http-client` service                                      | `netcoreapp3.1`                                                               |
-| **exporter**                    | Value for [`OTEL_TRACES_EXPORTER`](../docs/config.md#exporters) env. variable       | `otlp`                                                                        |
+| **tracesexporter**              | Value for [`OTEL_TRACES_EXPORTER`](../docs/config.md#exporters) env. variable       | `otlp`                                                                        |
```
Comment (Kielek) [inline examples/README.md:42]: Consider, whole PR.
tracesexporter -> tracesExporter
metricsexporter -> metricsExporter

**Review by Kielek — APPROVED** [review]

**Review by Kielek — APPROVED** [review]
